string sent_id
string language
string sentence
string speech_act
Frame[] frame_roles
Frame[] frame_words